quic/s2n-quic-transport/src/path/mtu.rs

```diff
@@ -322,6 +325,11 @@ impl Controller {
         self.plpmtu as usize
     }
 
+    /// Returns the maximum size any packet can reach
+    pub fn max_mtu(&self) -> MaxMtu {
+        self.max_mtu
+    }
```

Comment:
This would be max MTU inclusive of IP and UDP headers, but you're not encrypting the headers, so maybe you could go a little smaller? Could you use `max_udp_payload`?

Reply:
I thought about the `max_udp_payload` but that has the issue of starting with an IPv6 address and migrating to an IPv4 address. I could theoretically trim the IPv4 and UDP headers but that's only 28 bytes so i'm not sure that's even worth making the distinction.